simply_fit/simply_fit_api/views.py

```diff
@@ -11,13 +13,45 @@ def createUser(userName, firstName, lastName):
 			return "Success"
 		else:
 			raise KeyError("User exists, please use a different username")
+	
 	@staticmethod
 	def getUser(user):
 		try:
 			user = User.objects.get(userName=user)
 			return user
 		except:
 			return None
+
+	''' This method add the date and workout hours for a specified user
+		addDateAndWorkoutHours:
+			@params : username: [username; firstName; lastName]
+					  date: dateTime
```

Comment:
You are using a django.util.timezone as variable type in database. Please match type.